android/templates/module/default/template/android/manifest.ejs

```diff
@@ -15,4 +15,6 @@ name: <%- moduleName %>
 moduleid: <%- moduleId %>
 guid: <%- guid %>
 platform: <%- platform %>
+classpaths:
```

Comment:
Can you give an example how this would be used? I feel like this should solely be done in the module's gradle files, not the manifest - which is rather meant for top-level metadata.

Reply:
The easiest example and which actually led me to create this PR is that someone from our channel on slack asked if someone can port datadog for android, and when I check the setup for datadog, I found it like this:

```groovy
buildscript {
    dependencies {
        classpath("com.datadoghq:dd-sdk-android-gradle-plugin:x.x.x")
    }
}
plugins {
    id("com.datadoghq.dd-sdk-android-gradle-plugin")
    //(...)
}
android {
    //(...)
}
dependencies {
    implementation "com.datadoghq:dd-sdk-android-rum:x.x.x" 
    //(...)
}

```

You need to add a classpath, and you need to apply a plugin, hopefully this is sufficient answer.

Reply:
So how would a manifest file look in the above case?

Reply:
@AbdullahFaqeir Waiting for an example of the manifest, then we can merge

Reply:
classpaths: com.datadoghq:dd-sdk-android-gradle-plugin:x.x.x
plugins: com.datadoghq.dd-sdk-android-gradle-plugin

and the dependency would be used as usual

Reply:
> @hansemannn updated the manifest with how to use.